spek-samples/build.gradle

```diff
@@ -3,7 +3,7 @@ apply from: "$rootDir/gradle/common/kotlin.gradle"
 apply plugin: 'org.junit.platform.gradle.plugin'
 
 junitPlatform {
-    platformVersion '1.0.0-M3'
+    platformVersion '1.0.0-M4'
```

Comment:
Ok we definitely need to move this to separate `dependencies.gradle` file, too much duplication :) 

@raniejade I'll do this in a separate PR if you don't mind